### ISSUE TITLE
Installation problem "Cannot continue"

### ISSUE THREAD

**Issue by IfyOgbolu** [biolab/orange3#6514]
Creating an new conda env in "C:\Program Files\Orange"
Output folder: C:\Users\OGBOLU~1\AppData\Local\Temp\nsxC898.tmp\Orange-installer-data\conda-pkgs
Extract: _py-xgboost-mutex-2.0-cpu_0.tar.bz2
Extract: anyio-3.6.2-pyhd8ed1ab_0.tar.bz2
Extract: anyqt-0.2.0-pyh6c4a22f_0.tar.bz2
Extract: asttokens-2.2.1-pyhd8ed1ab_0.conda
Extract: backcall-0.2.0-pyh9f0ad1d_0.tar.bz2
Extract: backports-1.0-pyhd8ed1ab_3.conda
Extract: backports.functools_lru_cache-1.6.4-pyhd8ed1ab_0.tar.bz2
Extract: baycomp-1.0.2-py_1.tar.bz2
Extract: blas-2.116-openblas.tar.bz2
Extract: blas-devel-3.9.0-16_win64_openblas.tar.bz2
Extract: bottleneck-1.3.7-py39hc266a54_0.conda
Extract: brotli-1.0.9-hcfcfb64_8.tar.bz2
Extract: brotli-bin-1.0.9-hcfcfb64_8.tar.bz2
Extract: brotlipy-0.7.0-py39ha55989b_1005.tar.bz2
Extract: bzip2-1.0.8-h8ffe710_4.tar.bz2
Extract: ca-certificates-2022.12.7-h5b45459_0.conda
Extract: cachecontrol-0.12.11-pyhd8ed1ab_1.conda
Extract: cairo-1.16.0-hdecc03f_1015.conda
Extract: catboost-1.2-py39hcbf5309_4.conda
Extract: certifi-2022.12.7-pyhd8ed1ab_0.conda
Extract: cffi-1.15.1-py39h68f70e3_3.conda
Extract: chardet-5.1.0-py39hcbf5309_0.conda
Extract: charset-normalizer-3.1.0-pyhd8ed1ab_0.conda
Extract: colorama-0.4.6-pyhd8ed1ab_0.tar.bz2
Extract: commonmark-0.9.1-py_0.tar.bz2
Extract: conda-spec.txt
Extract: contourpy-1.0.7-py39h1f6ef14_0.conda
Extract: cryptography-40.0.2-py39hb6bd5e6_0.conda
Extract: cycler-0.11.0-pyhd8ed1ab_0.tar.bz2
Extract: debugpy-1.6.7-py39h99910a6_0.conda
Extract: decorator-5.1.1-pyhd8ed1ab_0.tar.bz2
Extract: dictdiffer-0.9.0-pyhd8ed1ab_0.tar.bz2
Extract: docutils-0.19-py39hcbf5309_1.tar.bz2
Extract: et_xmlfile-1.1.0-pyhd8ed1ab_0.conda
Extract: executing-1.2.0-pyhd8ed1ab_0.tar.bz2
Extract: expat-2.5.0-h63175ca_1.conda
Extract: font-ttf-dejavu-sans-mono-2.37-hab24e00_0.tar.bz2
Extract: font-ttf-inconsolata-3.000-h77eed37_0.tar.bz2
Extract: font-ttf-source-code-pro-2.038-h77eed37_0.tar.bz2
Extract: font-ttf-ubuntu-0.83-hab24e00_0.tar.bz2
Extract: fontconfig-2.14.2-hbde0cde_0.conda
Extract: fonts-conda-ecosystem-1-0.tar.bz2
Extract: fonts-conda-forge-1-0.tar.bz2
Extract: fonttools-4.39.3-py39ha55989b_0.conda
Extract: freetype-2.12.1-h546665d_1.conda
Extract: fribidi-1.0.10-h8d14728_0.tar.bz2
Extract: future-0.18.3-pyhd8ed1ab_0.conda
Extract: getopt-win32-0.1-h8ffe710_0.tar.bz2
Extract: gettext-0.21.1-h5728263_0.tar.bz2
Extract: glib-2.76.2-h12be248_0.conda
Extract: glib-tools-2.76.2-h12be248_0.conda
Extract: graphite2-1.3.13-1000.tar.bz2
Extract: graphviz-8.0.5-h51cb2cd_0.conda
Extract: gst-plugins-base-1.22.0-h001b923_2.conda
Extract: gstreamer-1.22.0-h6b5321d_2.conda
Extract: gts-0.7.6-h7c369d9_2.tar.bz2
Extract: h11-0.14.0-pyhd8ed1ab_0.tar.bz2
Extract: h2-4.1.0-py39hcbf5309_0.tar.bz2
Extract: harfbuzz-7.2.0-h196d34a_0.conda
Extract: hpack-4.0.0-pyh9f0ad1d_0.tar.bz2
Extract: httpcore-0.17.0-pyhd8ed1ab_0.conda
Extract: httpx-0.24.0-pyhd8ed1ab_1.conda
Extract: hyperframe-6.0.1-pyhd8ed1ab_0.tar.bz2
Extract: icu-72.1-h63175ca_0.conda
Extract: idna-3.4-pyhd8ed1ab_0.tar.bz2
Extract: importlib-metadata-6.6.0-pyha770c72_0.conda
Extract: importlib-resources-5.12.0-pyhd8ed1ab_0.conda
Extract: importlib_metadata-6.6.0-hd8ed1ab_0.conda
Extract: importlib_resources-5.12.0-pyhd8ed1ab_0.conda
Extract: install.bat
Extract: ipykernel-6.14.0-py39h832f523_0.tar.bz2
Extract: ipython-8.4.0-py39hcbf5309_0.tar.bz2
Extract: ipython_genutils-0.2.0-py_1.tar.bz2
Extract: jaraco.classes-3.2.3-pyhd8ed1ab_0.tar.bz2
Extract: jedi-0.18.2-pyhd8ed1ab_0.conda
Extract: joblib-1.2.0-pyhd8ed1ab_0.tar.bz2
Extract: jupyter_client-8.2.0-pyhd8ed1ab_0.conda
Extract: jupyter_core-5.3.0-py39hcbf5309_0.conda
Extract: keyring-23.13.1-py39hcbf5309_0.conda
Extract: keyrings.alt-4.2.0-pyhd8ed1ab_0.conda
Extract: kiwisolver-1.4.4-py39h1f6ef14_1.tar.bz2
Extract: krb5-1.20.1-heb0366b_0.conda
Extract: lcms2-2.15-h3e3b177_1.conda
Extract: lerc-4.0.0-h63175ca_0.tar.bz2
Extract: libblas-3.9.0-16_win64_openblas.tar.bz2
Extract: libbrotlicommon-1.0.9-hcfcfb64_8.tar.bz2
Extract: libbrotlidec-1.0.9-hcfcfb64_8.tar.bz2
Extract: libbrotlienc-1.0.9-hcfcfb64_8.tar.bz2
Extract: libcblas-3.9.0-16_win64_openblas.tar.bz2
Extract: libclang-16.0.3-default_h8b4101f_0.conda
Extract: libclang13-16.0.3-default_h45d3cf4_0.conda
Extract: libdeflate-1.18-hcfcfb64_0.conda
Extract: libexpat-2.5.0-h63175ca_1.conda
Extract: libffi-3.4.2-h8ffe710_5.tar.bz2
Extract: libflang-5.0.0-h6538335_20180525.tar.bz2
Extract: libgd-2.3.3-h2ed9e1d_6.conda
Extract: libglib-2.76.2-he8f3873_0.conda
Extract: libiconv-1.17-h8ffe710_0.tar.bz2
Extract: libjpeg-turbo-2.1.5.1-hcfcfb64_0.conda
Extract: liblapack-3.9.0-16_win64_openblas.tar.bz2
Extract: liblapacke-3.9.0-16_win64_openblas.tar.bz2
Extract: libogg-1.3.4-h8ffe710_1.tar.bz2
Extract: libopenblas-0.3.21-pthreads_h02691f0_0.tar.bz2
Extract: libpng-1.6.39-h19919ed_0.conda
Extract: libsodium-1.0.18-h8d14728_1.tar.bz2
Extract: libsqlite-3.40.0-hcfcfb64_1.conda
Extract: libtiff-4.5.0-h6c8260b_6.conda
Extract: libvorbis-1.3.7-h0e60522_0.tar.bz2
Extract: libwebp-1.3.0-hcfcfb64_0.conda
Extract: libwebp-base-1.3.0-hcfcfb64_0.conda
Extract: libxcb-1.13-hcd874cb_1004.tar.bz2
Extract: libxgboost-1.7.4-cpu_h20390bd_0.conda
Extract: libxml2-2.10.4-hc3477c8_0.conda
Extract: libzlib-1.2.13-hcfcfb64_4.tar.bz2
Extract: llvm-meta-5.0.0-0.tar.bz2
Extract: lockfile-0.12.2-py_1.tar.bz2
Extract: m2w64-gcc-libgfortran-5.3.0-6.tar.bz2
Extract: m2w64-gcc-libs-5.3.0-7.tar.bz2
Extract: m2w64-gcc-libs-core-5.3.0-7.tar.bz2
Extract: m2w64-gmp-6.1.0-2.tar.bz2
Extract: m2w64-libwinpthread-git-5.0.0.4634.697f757-2.tar.bz2
Extract: matplotlib-base-3.7.1-py39haf65ace_0.conda
Extract: matplotlib-inline-0.1.6-pyhd8ed1ab_0.tar.bz2
Extract: more-itertools-9.1.0-pyhd8ed1ab_0.conda
Extract: msgpack-python-1.0.5-py39h1f6ef14_0.conda
Extract: msys2-conda-epoch-20160418-1.tar.bz2
Extract: munkres-1.1.4-pyh9f0ad1d_0.tar.bz2
Extract: nest-asyncio-1.5.6-pyhd8ed1ab_0.tar.bz2
Extract: networkx-3.1-pyhd8ed1ab_0.conda
Extract: numpy-1.24.3-py39h816b6a6_0.conda
Extract: openblas-0.3.21-pthreads_ha35c500_0.tar.bz2
Extract: openjpeg-2.5.0-ha2aaf27_2.conda
Extract: openmp-5.0.0-vc14_1.tar.bz2
Extract: openpyxl-3.1.2-py39ha55989b_0.conda
Extract: openssl-3.1.0-hcfcfb64_3.conda
Extract: opentsne-0.7.1-py39hbd792c9_0.conda
Extract: orange-canvas-core-0.1.31-pyhd8ed1ab_0.conda
Extract: orange-widget-base-4.21.0-pyhd8ed1ab_0.conda
Extract: orange3-3.35.0-py39h1679cfb_0.conda
Extract: packaging-23.1-pyhd8ed1ab_0.conda
Extract: pandas-1.5.3-py39h2ba5b7c_1.conda
Extract: pango-1.50.14-hd64ce24_1.conda
Extract: parso-0.8.3-pyhd8ed1ab_0.tar.bz2
Extract: pcre2-10.40-h17e33f8_0.tar.bz2
Extract: pickleshare-0.7.5-py_1003.tar.bz2
Extract: pillow-9.5.0-py39haa1d754_0.conda
Extract: pip-23.1.2-pyhd8ed1ab_0.conda
Extract: pixman-0.40.0-h8ffe710_0.tar.bz2
Extract: platformdirs-3.5.0-pyhd8ed1ab_0.conda
Extract: plotly-5.14.1-pyhd8ed1ab_0.conda
Extract: ply-3.11-py_1.tar.bz2
Extract: pooch-1.7.0-pyha770c72_3.conda
Extract: prompt-toolkit-3.0.38-pyha770c72_0.conda
Extract: psutil-5.9.5-py39ha55989b_0.conda
Extract: pthread-stubs-0.4-hcd874cb_1001.tar.bz2
Extract: pure_eval-0.2.2-pyhd8ed1ab_0.tar.bz2
Extract: py-xgboost-1.7.4-cpu_py39ha538f94_0.conda
Extract: pycparser-2.21-pyhd8ed1ab_0.tar.bz2
Extract: pygments-2.15.1-pyhd8ed1ab_0.conda
Extract: pyopenssl-23.1.1-pyhd8ed1ab_0.conda
Extract: pyparsing-3.0.9-pyhd8ed1ab_0.tar.bz2
Extract: pyqt-5.15.7-py39hb77abff_3.conda
Extract: pyqt5-sip-12.11.0-py39h99910a6_3.conda
Extract: pyqtgraph-0.13.3-pyhd8ed1ab_0.conda
Extract: pyqtwebengine-5.15.7-py39h2f4a3f1_3.conda
Extract: pysocks-1.7.1-py39hcbf5309_5.tar.bz2
Extract: python-3.9.12-hcf16a7b_1_cpython.tar.bz2
Extract: python-dateutil-2.8.2-pyhd8ed1ab_0.tar.bz2
Extract: python-graphviz-0.20.1-pyh22cad53_0.tar.bz2
Extract: python-louvain-0.16-pyhd8ed1ab_0.conda
Extract: python_abi-3.9-3_cp39.conda
Extract: pytz-2023.3-pyhd8ed1ab_0.conda
Extract: pywin32-304-py39h99910a6_2.tar.bz2
Extract: pywin32-ctypes-0.2.0-py39hcbf5309_1006.tar.bz2
Extract: pyyaml-6.0-py39ha55989b_5.tar.bz2
Extract: pyzmq-25.0.2-py39hea35a22_0.conda
Extract: qasync-0.24.0-pyhd8ed1ab_0.conda
Extract: qt-main-5.15.8-h7f2b912_9.conda
Extract: qt-webengine-5.15.8-h5b1ea0b_0.tar.bz2
Extract: qtconsole-5.4.3-pyhd8ed1ab_0.conda
Extract: qtconsole-base-5.4.3-pyha770c72_0.conda
Extract: qtpy-2.3.1-pyhd8ed1ab_0.conda
Extract: requests-2.29.0-pyhd8ed1ab_0.conda
Extract: scikit-learn-1.1.3-py39h6fe01c0_1.tar.bz2
Extract: scipy-1.10.1-py39hde5eda1_1.conda
Extract: serverfiles-0.3.0-py_0.tar.bz2
Extract: setuptools-67.7.2-pyhd8ed1ab_0.conda
Extract: sip-6.7.9-py39h99910a6_0.conda
Extract: sitecustomize.py
Extract: six-1.16.0-pyh6c4a22f_0.tar.bz2
Extract: sniffio-1.3.0-pyhd8ed1ab_0.tar.bz2
Extract: sqlite-3.40.0-hcfcfb64_1.conda
Extract: stack_data-0.6.2-pyhd8ed1ab_0.conda
Extract: tenacity-8.2.2-pyhd8ed1ab_0.conda
Extract: threadpoolctl-3.1.0-pyh8a188c0_0.tar.bz2
Extract: tk-8.6.12-h8ffe710_0.tar.bz2
Extract: toml-0.10.2-pyhd8ed1ab_0.tar.bz2
Extract: tomli-2.0.1-pyhd8ed1ab_0.tar.bz2
Extract: tornado-6.3-py39ha55989b_0.conda
Extract: traitlets-5.9.0-pyhd8ed1ab_0.conda
Extract: typing-extensions-4.5.0-hd8ed1ab_0.conda
Extract: typing_extensions-4.5.0-pyha770c72_0.conda
Extract: tzdata-2023c-h71feb2d_0.conda
Extract: ucrt-10.0.22621.0-h57928b3_0.tar.bz2
Extract: unicodedata2-15.0.0-py39ha55989b_0.tar.bz2
Extract: urllib3-1.26.15-pyhd8ed1ab_0.conda
Extract: vc-14.3-hb25d44b_16.conda
Extract: vc14_runtime-14.34.31931-h5081d32_16.conda
Extract: vs2015_runtime-14.34.31931-hed1258a_16.conda
Extract: wcwidth-0.2.6-pyhd8ed1ab_0.conda
Extract: wheel-0.40.0-pyhd8ed1ab_0.conda
Extract: win_inet_pton-1.1.0-py39hcbf5309_5.tar.bz2
Extract: xgboost-1.7.4-cpu_py39ha538f94_0.conda
Extract: xlrd-2.0.1-pyhd8ed1ab_3.tar.bz2
Extract: xlsxwriter-3.1.0-pyhd8ed1ab_0.conda
Extract: xorg-kbproto-1.0.7-hcd874cb_1002.tar.bz2
Extract: xorg-libice-1.0.10-hcd874cb_0.tar.bz2
Extract: xorg-libsm-1.2.3-hcd874cb_1000.tar.bz2
Extract: xorg-libx11-1.8.4-hcd874cb_0.conda
Extract: xorg-libxau-1.0.9-hcd874cb_0.tar.bz2
Extract: xorg-libxdmcp-1.1.3-hcd874cb_0.tar.bz2
Extract: xorg-libxext-1.3.4-hcd874cb_2.conda
Extract: xorg-libxpm-3.5.13-hcd874cb_0.tar.bz2
Extract: xorg-libxt-1.2.1-hcd874cb_2.tar.bz2
Extract: xorg-xextproto-7.3.0-hcd874cb_1003.conda
Extract: xorg-xproto-7.0.31-hcd874cb_1007.tar.bz2
Extract: xz-5.2.6-h8d14728_0.tar.bz2
Extract: yaml-0.2.5-h8ffe710_2.tar.bz2
Extract: zeromq-4.3.4-h0e60522_1.tar.bz2
Extract: zipp-3.15.0-pyhd8ed1ab_0.conda
Extract: zlib-1.2.13-hcfcfb64_4.tar.bz2
Extract: zstd-1.5.2-h12be248_6.conda
Output folder: C:\Users\OGBOLU~1\AppData\Local\Temp\nsxC898.tmp\Orange-installer-data\conda-pkgs
Installing packages (this might take a while)
Executing: cmd.exe /c install.bat "C:\Program Files\Orange" "C:\Users\Ogbolu Ify\miniconda3\condabin\conda.bat"
Creating a conda env in "C:\Program Files\Orange"
failed to create process.
Appending conda-forge channel
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
failed to create process.
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
        0 file(s) copied.
"conda" command exited with 1. Cannot continue.

**What's wrong?**

**How can we reproduce the problem?** 

**What's your environment?**

- Operating system:
- Orange version:
- How you installed Orange:

**Comment by thocevar** [comment]
The installer reports (during installation of miniconda) that the destination folder contains an invalid character '\~'. Tilde in your username is the probable culprit for this problem.

**Comment by thocevar** [comment]
Or it could be the space in the username. In this case (miniconda) installer also warns that spaces in usernames and therefore paths can cause problems with several Conda packages.

**Comment by IfyOgbolu** [comment]
Thanks Tomaž for the feedback. I sorted this  issue already.
